code/controllers/subsystem/vote.dm

```diff
@@ -192,34 +204,38 @@ SUBSYSTEM_DEF(vote)
 			if("gamemode")
 				choices.Add(config.votable_modes)
 			if("map")
-				if(!admin && SSmapping.map_voted)
-					to_chat(usr, "<span class='warning'>The next map has already been selected.</span>")
-					return FALSE
-				for(var/map in config.maplist)
+				// Randomizes the list so it isn't always METASTATION
+				var/list/maps = list()
+				for(var/map in global.config.maplist)
 					var/datum/map_config/VM = config.maplist[map]
 					if(!VM.votable || (VM.map_name in SSpersistence.blocked_maps))
 						continue
-					choices.Add(VM.map_name)
+					maps += VM.map_name
+					shuffle_inplace(maps)
+				for(var/valid_map in maps)
+					choices.Add(valid_map)
+			if("transfer")
+				choices.Add("Initiate Crew Transfer", "Continue Playing")
```

Comment:
Why the changes here?

Reply:
This was a change I implemented long ago with the original vote panel. If users left the vote panel up it would (or close it) it would vote for the first option, thus a huge chunk of votes would always go to metastation etc. This shuffles the list of maps.
It's not necessary now in particular and it doesn't seem harmful either, but I could remove it if you'd like